.github/workflows/ci.yml

```diff
@@ -27,9 +27,9 @@ jobs:
       # A GitHub action to setup Node.js
       - uses: actions/setup-node@v1
         with:
-          node-version: '14'
+          node-version: "14"
 
       # Run the NPM install command before proceeding
       - run: npm install
 
-      - run: npm run lint
\ No newline at end of file
+      - run: npm run lint
```

Comment:
YESSSS!

Reply:
We may want to include a coverage report in the future. But this is a great addition for now.